src/Xamarin.Android.Sdk/targets/Xamarin.Android.Sdk.props

```diff
@@ -32,14 +32,16 @@
 
   <!--  User-facing configuration-specific defaults -->
   <PropertyGroup Condition=" '$(Configuration)' == 'Debug' ">
-    <AndroidUseSharedRuntime Condition=" '$(AndroidUseSharedRuntime)' == '' ">true</AndroidUseSharedRuntime>
+    <!--FIXME: Disable the shared runtime by default -->
+    <AndroidUseSharedRuntime Condition=" '$(AndroidUseSharedRuntime)' == '' ">false</AndroidUseSharedRuntime>
```

Comment:
Also default `$(EmbedAssembliesIntoApk)` to `true` for .NET 5 projects, so we won't be using Fast Deployment yet at all.